product_pricelist_simulation/wizards/wizard_preview_pricelist_views.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_pricelist_simulation_view_form" model="ir.ui.view">
+        <field name="name">wizard.preview.pricelist.form</field>
+        <field name="model">wizard.preview.pricelist</field>
+        <field name="arch" type="xml">
+            <form>
+                <field name="variant_count" invisible="1" />
+                <group name="settings_group">
+                    <group name="settings_left_group">
+                        <field name="template_id" />
+                        <field name="product_id" options="{'no_create': True}" />
```

Comment:
```suggestion
                        <field name="product_id" options="{'no_create': True}" attrs="{'invisible': [('variant_count', '<', 2)]}"/>
```

I think this field is useless in mono variant product template and can be hidden.